<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ToolBar</class>
 <widget class="QMainWindow" name="ToolBar">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>829</width>
    <height>662</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="iconSize">
   <size>
    <width>49</width>
    <height>40</height>
   </size>
  </property>
  <property name="animated">
   <bool>true</bool>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="Cube">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>40</y>
      <width>371</width>
      <height>401</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Choisir Plan Parallele du plan &amp;quot;ABEF&amp;quot;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="ressource.qrc">:/icone/cube.gif</pixmap>
    </property>
   </widget>
   <widget class="QPushButton" name="Horizontal">
    <property name="geometry">
     <rect>
      <x>630</x>
      <y>140</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Choisir Plan Parallele du plan &amp;quot;EFGH&amp;quot;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="autoFillBackground">
     <bool>true</bool>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="ressource.qrc">
      <normaloff>:/icone/bas.png</normaloff>:/icone/bas.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>50</width>
      <height>50</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="VerticalGD">
    <property name="geometry">
     <rect>
      <x>710</x>
      <y>210</y>
      <width>41</width>
      <height>31</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Choisir Plan Parallele du plan &amp;quot;BCFG&amp;quot;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="ressource.qrc">
      <normaloff>:/icone/GAUCHE.png</normaloff>:/icone/GAUCHE.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>40</width>
      <height>35</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="VerticalDA">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>240</y>
      <width>31</width>
      <height>41</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Choisir Plan Parallele du plan &amp;quot;ABEF&amp;quot;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="ressource.qrc">
      <normaloff>:/icone/AR.png</normaloff>:/icone/AR.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>40</width>
      <height>40</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="Plan1">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>50</y>
      <width>101</width>
      <height>101</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Plan 1&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="ressource.qrc">
      <normaloff>:/icone/VAA1.png</normaloff>:/icone/VAA1.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>110</width>
      <height>100</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="Plan3">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>50</y>
      <width>91</width>
      <height>101</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="ressource.qrc">
      <normaloff>:/icone/VAA3.png</normaloff>:/icone/VAA3.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>100</width>
      <height>100</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="Plan2">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>50</y>
      <width>91</width>
      <height>101</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="ressource.qrc">
      <normaloff>:/icone/VAA2.png</normaloff>:/icone/VAA2.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>100</width>
      <height>100</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="Plan7">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>280</y>
      <width>101</width>
      <height>101</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Plan 1&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="ressource.qrc">
      <normaloff>:/icone/VAA7.png</normaloff>:/icone/VAA7.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>110</width>
      <height>100</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="Plan6">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>170</y>
      <width>101</width>
      <height>101</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Plan 1&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="ressource.qrc">
      <normaloff>:/icone/VAA6.png</normaloff>:/icone/VAA6.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>110</width>
      <height>100</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="Plan9">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>280</y>
      <width>101</width>
      <height>101</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Plan 1&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="ressource.qrc">
      <normaloff>:/icone/VAA9.png</normaloff>:/icone/VAA9.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>110</width>
      <height>100</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="Plan8">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>280</y>
      <width>101</width>
      <height>101</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Plan 1&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="ressource.qrc">
      <normaloff>:/icone/VAA8.png</normaloff>:/icone/VAA8.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>110</width>
      <height>100</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="Plan5">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>170</y>
      <width>101</width>
      <height>101</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Plan 1&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="ressource.qrc">
      <normaloff>:/icone/VAA5.png</normaloff>:/icone/VAA5.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>110</width>
      <height>100</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="Plan4">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>170</y>
      <width>101</width>
      <height>101</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Plan 1&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="ressource.qrc">
      <normaloff>:/icone/VAA4.png</normaloff>:/icone/VAA4.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>110</width>
      <height>100</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="Plan7_2">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>180</y>
      <width>31</width>
      <height>101</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Plan 1&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="ressource.qrc">
      <normaloff>:/icone/VDG7.png</normaloff>:/icone/VDG7.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>110</width>
      <height>100</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="Plan6_2">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>180</y>
      <width>31</width>
      <height>101</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Plan 1&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="ressource.qrc">
      <normaloff>:/icone/VDG6.png</normaloff>:/icone/VDG6.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>110</width>
      <height>100</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="Plan3_2">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>180</y>
      <width>31</width>
      <height>101</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="ressource.qrc">
      <normaloff>:/icone/VDG3.png</normaloff>:/icone/VDG3.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>100</width>
      <height>100</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="Plan5_2">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>180</y>
      <width>21</width>
      <height>101</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Plan 1&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="ressource.qrc">
      <normaloff>:/icone/VDG5.png</normaloff>:/icone/VDG5.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>110</width>
      <height>100</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="Plan9_2">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>180</y>
      <width>31</width>
      <height>101</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Plan 1&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="ressource.qrc">
      <normaloff>:/icone/VDG9.png</normaloff>:/icone/VDG9.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>110</width>
      <height>100</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="Plan4_2">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>180</y>
      <width>31</width>
      <height>101</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Plan 1&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="ressource.qrc">
      <normaloff>:/icone/VDG4.png</normaloff>:/icone/VDG4.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>110</width>
      <height>100</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="Plan8_2">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>180</y>
      <width>31</width>
      <height>101</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Plan 1&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="ressource.qrc">
      <normaloff>:/icone/VDG8.png</normaloff>:/icone/VDG8.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>110</width>
      <height>100</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="Plan1_2">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>180</y>
      <width>31</width>
      <height>101</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Plan 1&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="ressource.qrc">
      <normaloff>:/icone/VDG1.png</normaloff>:/icone/VDG1.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>10000</width>
      <height>100</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="Plan2_2">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>180</y>
      <width>31</width>
      <height>101</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="ressource.qrc">
      <normaloff>:/icone/VDG2.png</normaloff>:/icone/VDG2.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>100</width>
      <height>100</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="Plan7_3">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>320</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Plan 1&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="ressource.qrc">
      <normaloff>:/icone/H7.png</normaloff>:/icone/H7.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>110</width>
      <height>100</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="Plan6_3">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>280</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Plan 1&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="ressource.qrc">
      <normaloff>:/icone/H6.png</normaloff>:/icone/H6.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>110</width>
      <height>100</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="Plan3_3">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>160</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="ressource.qrc">
      <normaloff>:/icone/H3.png</normaloff>:/icone/H3.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>100</width>
      <height>100</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="Plan5_3">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>240</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Plan 1&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="ressource.qrc">
      <normaloff>:/icone/H5.png</normaloff>:/icone/H5.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>110</width>
      <height>100</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="Plan9_3">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>400</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Plan 1&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="ressource.qrc">
      <normaloff>:/icone/H9.png</normaloff>:/icone/H9.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>110</width>
      <height>100</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="Plan4_3">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>200</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Plan 1&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="ressource.qrc">
      <normaloff>:/icone/H4.png</normaloff>:/icone/H4.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>110</width>
      <height>100</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="Plan8_3">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>360</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Plan 1&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="ressource.qrc">
      <normaloff>:/icone/H8.png</normaloff>:/icone/H8.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>110</width>
      <height>100</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="Plan1_3">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>80</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Plan 1&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="ressource.qrc">
      <normaloff>:/icone/H1.png</normaloff>:/icone/H1.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>110</width>
      <height>100</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="Plan2_3">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>120</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset resource="ressource.qrc">
      <normaloff>:/icone/H2.png</normaloff>:/icone/H2.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>100</width>
      <height>100</height>
     </size>
    </property>
   </widget>
   <zorder>Plan9</zorder>
   <zorder>Plan8</zorder>
   <zorder>Plan7</zorder>
   <zorder>Plan6</zorder>
   <zorder>Plan5</zorder>
   <zorder>Plan4</zorder>
   <zorder>Cube</zorder>
   <zorder>VerticalGD</zorder>
   <zorder>VerticalDA</zorder>
   <zorder>Horizontal</zorder>
   <zorder>Plan3</zorder>
   <zorder>Plan2</zorder>
   <zorder>Plan1</zorder>
   <zorder>Plan7_2</zorder>
   <zorder>Plan6_2</zorder>
   <zorder>Plan3_2</zorder>
   <zorder>Plan5_2</zorder>
   <zorder>Plan9_2</zorder>
   <zorder>Plan4_2</zorder>
   <zorder>Plan8_2</zorder>
   <zorder>Plan1_2</zorder>
   <zorder>Plan2_2</zorder>
   <zorder>Plan7_3</zorder>
   <zorder>Plan6_3</zorder>
   <zorder>Plan3_3</zorder>
   <zorder>Plan5_3</zorder>
   <zorder>Plan9_3</zorder>
   <zorder>Plan4_3</zorder>
   <zorder>Plan8_3</zorder>
   <zorder>Plan1_3</zorder>
   <zorder>Plan2_3</zorder>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <property name="layoutDirection">
    <enum>Qt::RightToLeft</enum>
   </property>
   <property name="autoFillBackground">
    <bool>true</bool>
   </property>
   <property name="iconSize">
    <size>
     <width>30</width>
     <height>30</height>
    </size>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionDupliquer"/>
   <addaction name="actionSelectionner"/>
  </widget>
  <action name="actionSelectionner">
   <property name="icon">
    <iconset resource="ressource.qrc">
     <normaloff>:/icone/Selection_Plan.jpeg</normaloff>
     <normalon>fleche.jpeg</normalon>:/icone/Selection_Plan.jpeg</iconset>
   </property>
   <property name="text">
    <string>Selectionner</string>
   </property>
   <property name="toolTip">
    <string>Selectionner les LEDs</string>
   </property>
  </action>
  <action name="actionDupliquer">
   <property name="icon">
    <iconset resource="ressource.qrc">
     <normaloff>:/icone/Dupliquer.gif</normaloff>:/icone/Dupliquer.gif</iconset>
   </property>
   <property name="text">
    <string>Dupliquer</string>
   </property>
   <property name="toolTip">
    <string>Dupliquer</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="ressource.qrc"/>
 </resources>
 <connections/>
</ui>
